presto-main/src/main/java/com/facebook/presto/operator/aggregation/ApproximateAverageAggregations.java

```diff
@@ -79,15 +79,15 @@ public static void combine(ApproximateAverageState state, ApproximateAverageStat
         }
     }
 
-    @OutputFunction(StandardTypes.VARCHAR)
+    @OutputFunction("varchar(57)")
     public static void output(ApproximateAverageState state, double confidence, BlockBuilder out)
     {
         if (state.getCount() == 0) {
             out.appendNull();
         }
         else {
             String result = formatApproximateAverage(state.getSamples(), state.getMean(), state.getM2() / state.getCount(), confidence);
-            VARCHAR.writeString(out, result);
+            createUnboundedVarcharType().writeString(out, result);
```

Comment:
This doesn't look right. The signature declares `varchar(57)`, but you're writing with the unbound type. I think you need to use `VarcharType.createVarcharType`